Core/FeatureFlag.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  FeatureFlag.swift
+//  DuckDuckGo
+//
+//  Copyright © 2023 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+import BrowserServicesKit
+
+public enum FeatureFlag: String {
+    case debugMenu
+    case sync
+    case credentialsAutofill
+    case credentialsSaving
+    case inlineIconCredentials
+    case accessCredentialManagement
+}
+
+extension FeatureFlag: FeatureFlagSourceProviding {
+    public var source: FeatureFlagSource {
+        switch self {
+        case .debugMenu, .sync, .credentialsAutofill, .credentialsSaving, .inlineIconCredentials, .accessCredentialManagement:
+            return .internalOnly
```

Comment:
As we now have the Autofill features merged to the Remote Config as internal-only, does it make sense to use `remoteDevelopment` here for the applicable subfeatures?

Reply:
Of course: https://github.com/duckduckgo/iOS/pull/1565 🙂

Reply:
Exactly :) Until the `internal` state handling is added to the Privacy Config, that would result in autofill being either off or on for all users. So makes sense to promote it in that PR.